azext_iot/_help.py

```diff
@@ -331,10 +331,10 @@
     short-summary: Create an IoT Edge deployment in the target IoT Hub.
     long-summary: Configuration json must have root of 'content' like {"content":{...}}.
     examples:
-    - name: Create deployment with condition where a device is in 'building 9' and 
+    - name: Create deployment with condition where a device is in 'building 9' and
             the environment is 'test'.
       text: >
-        az iot edge deployment create -c myconfig -n myhub --content ../mycontent.json 
+        az iot edge deployment create -c myconfig -n myhub --content ../mycontent.json
```

Comment:
We might as well variable'ize the deployment config id and hub parameters for the edge deployment <> help for consistency. Something Like:

az iot edge deployment create -c [Config Id] -n [IoTHub Name] --content ../mycontent.json

Reply:
sure why not